api/types.go

```diff
@@ -437,11 +438,12 @@ func PinCid(c *cid.Cid) Pin {
 
 // PinSerial is a serializable version of Pin
 type PinSerial struct {
-	Cid               string   `json:"cid"`
-	Name              string   `json:"name"`
-	Allocations       []string `json:"allocations"`
-	Everywhere        bool     `json:"everywhere,omitempty"` // legacy
-	ReplicationFactor int      `json:"replication_factor"`
+	Cid                  string   `json:"cid"`
+	Name                 string   `json:"name"`
+	Allocations          []string `json:"allocations"`
+	ReplicationFactor    int      `json:"replication_factor,omitempty"` //legacy
```

Comment:
To be honest, since we have state migration, it might just be ok to remove it. We don't have a policy, but communication between nodes running different versions can't happen because the libp2p streams are tagged with cluster version.